### ISSUE TITLE
Missing supplementary materials in the updated preprint?

### ISSUE THREAD

**Issue by naoto0804** [posterllava/PosterLLaVA#11]
Hi, what a nice project! I am astonished by how the generation process is further simplified.
I'm wondering how strong the `WCAG standard-based color adjusting scheme` is by re-implementing it on my own, but I can't find any detailed descriptions because of missing supplementary materials in the v3 manuscript. Is there any plan to release the supplementary material?

(by the way, I'm the first author of OpenCOLE. Thank you for citing OpenCOLE.)
